sklearn/decomposition/_dict_learning.py

```diff
@@ -1378,6 +1408,21 @@ class MiniBatchDictionaryLearning(SparseCodingMixin, BaseEstimator):
         RandomState instance that is generated either from a seed, the random
         number generattor or by `np.random`.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.datasets import make_sparse_coded_signal
+    >>> from sklearn.decomposition import MiniBatchDictionaryLearning
+    >>>
+    >>> np.random.seed(42)
+    >>> X, dictionary, code = make_sparse_coded_signal(
+    ... n_samples=100, n_components=15, n_features=20, n_nonzero_coefs=10)
+    >>> dico = MiniBatchDictionaryLearning(n_components=15,
```

Comment:
I am not sure `dico` is a good name. It's a French shorthand for dictionary but I am not sure in makes sense in English. Maybe just `est` (for estimator) or `dict_learner` would make more sense.

Reply:
Done this